app/frontend/src/components/BotsettingsDrawer/BotsettingsDrawer.tsx

```
@@ -91,188 +90,217 @@ export const BotsettingsDrawer = ({
-                                rows={3}
+                                rows={15}
                                 onChange={onDescriptionChange}
                             />
-                        </Field>
-                    </div>
+                        ) : (
+                            <Markdown
+                                className={styles.markdownDescription}
+                                remarkPlugins={[remarkGfm]}
+                                rehypePlugins={[rehypeRaw]}
+                                children={description}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)